ITC205_BaseLine_Code/src/ReturnBookControl.java

```diff
@@ -1,31 +1,27 @@
 public class ReturnBookControl {
 
 	private ReturnBookUI ui;

```

Comment:
D12 - Change:
        private ReturnBookUI ui;
	private enum ReturnBookControlState { INITIALISED, READY, INSPECTING };
	
	private ReturnBookControlState state;
	private library library;
	private loan currentLoan;

To:

	private enum ReturnBookControlState { INITIALISED, READY, INSPECTING };
	
        private ReturnBookUI ui;
	private ReturnBookControlState state;
	private library library;
	private loan currentLoan;